releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,5 +1,13 @@
 ## Microsoft Extensions Azure App Configuration
 
+### v2.0.0-preview-009200001-1437 - July 10, 2019
+* Replaced watch feature with on-demand refresh mechanism
+    * Watch feature had issues that resulted in too many server requests to keep the configuration updated
```

Comment:
I think this might be a little vague for those looking back and wondering why watch was abandoned. Also the wording makes it sound like there was an issue in `Watch` that could have just been fixed rather than the fact that it was fundamentally an undesirable design that we moved away from. I would like to be super informative for this one. How about 

"The watch mechanism had some fundamental design flaws. It was not able to be invoked on-demand. It was always running even in applications that could be considered inactive. It promoted constant polling of configuration rather than a more intelligent approach of refreshing configuration when applications are active or need to ensure freshness"